api/src/search/search.manager.js

```diff
@@ -39,12 +39,43 @@ export async function searchParams(args) {
         if(results.rows.length===1){
             const link = results.rows[0];
             params = link.long
-            // console.log("--------PARAMS-----", params)
-
         }
-        // console.log("___________>>>>SERACH PARAMS<<<<<__________", params)
+
+        let parsedParams = JSON.parse(params);
+        let aggFilter =[];
+        parsedParams['agg_filters'].map((agg,index)=>{
+            let tempAgg = {};
+            for (const [key, value] of Object.entries(agg)) {
+                if(key=='include_missing_fields'){
+                    tempAgg['includeMissingFields'] = value;
+                }else{
+                    tempAgg[key]=value;
+                }
+            }
+            aggFilter.push(tempAgg)
+        });
+        parsedParams['agg_filters']=aggFilter;
+
+        let crowdAggFilter=[];
+        parsedParams['crowd_agg_filters'].map((agg,index)=>{
+            let tempAgg = {};
+
+            for (const [key, value] of Object.entries(agg)) {
+                if(key=='include_missing_fields'){
```

Comment:
Curious why we would use mix of camel and snake at all. Isn’t this field fully defined in front end so should all match and still possible error or is there more refactoring of this whole flow of data to further clarify how islands and facets and hash/parameters are translated and pushed around.